src/main/web/templates/handlebars/partials/json-ld/base.handlebars

```diff
@@ -0,0 +1,15 @@
+{{#if_any (eq type "home_page") (eq type "timeseries") (eq type "bulletin") (eq type "article") (eq type "static_adhoc")}}
+    <script type="application/ld+json">
+        {
+            "@context": "http://schema.org",
+            "name": "{{description.title}}",
+            "description": {{#if description._abstract}}"{{description._abstract}}"{{else}}"{{description.summary}}"{{/if}},
```

Comment:
Trailing comma needs to be removed
```suggestion
            "description": {{#if description._abstract}}"{{description._abstract}}"{{else}}"{{description.summary}}"{{/if}}
```

Reply:
I think this one is valid, unless i'm missing something?